packages/shared/src/permissions/permissionsClass.ts

```diff
@@ -728,11 +728,22 @@ export class Permissions {
   public canReadMultiProjectResource = (
     projects: string[] | undefined
   ): boolean => {
-    // If the resource is available to all projects (an empty array or no projects property), then everyone should have read access
-    if (!projects || !projects.length || this.superAdmin) {
+    if (this.superAdmin) {
       return true;
     }
 
+    // If the resource doesn't have a projects property or it's an empty array
+    // that means it's in all projects
+    if (!projects || !projects.length) {
+      // If the user has read access via their global role, they should be able to read
+      if (this.hasPermission("readData", "")) {
+        return true;
+      }
+      // if the user has read access globally, or in atleast 1 project they should have read access
+      return this.hasReadAccessForAtleast1Project();
+    }
```

Comment:
I think we can simplify this a lot.  No need for a separate `hasReadAccessForAtLeast1Project` method.

```ts
    if (!projects || !projects.length) {
      // Must have read access globally or in at least 1 project
      const projectsToCheck = [
        "",
        ...Object.keys(this.userPermissions.projects),
      ];
      return projectsToCheck.some((p) => this.hasPermission("readData", p));
    }
```